Demo/src/main/java/com/paypal/android/api/services/SDKSampleServerAPI.kt

```diff
@@ -136,7 +128,7 @@ class SDKSampleServerAPI {
         if (DEFAULT_ORDER_ID != null) {
             Order(DEFAULT_ORDER_ID, "CREATED")
         } else {
-            val body = JsonParser.parseString(orderRequest.toString()) as JsonObject
```

Comment:
What is your reason for changing this?

Reply:
Thanks for mentioning this. I may investigate it a bit further. What I think happened is the removal of mxo caused a ripple effect that downgraded the version of GSON linked by the Demo app somehow.

I'm going to upgrade the explicit version of GSON we reference in `build.gradle` to see if we can keep `JsonParser.parseString()` like it was before.

Reply:
Was able to clean it up! [d46b91f](https://github.com/paypal/paypal-android/pull/275/commits/d46b91f683f486a82af38c086db57a15967280eb)

Reply:
Great work! I didn't know that it does so much automatically.

Reply:
Yeah it has to do with dependency resolution in [Gradle](https://docs.gradle.org/current/userguide/dependency_resolution.html#sub:resolution-strategy). When there are two versions of the same library linked to a project, Gradle will always pick the newest one.